build/merge.py

```diff
@@ -13,6 +9,10 @@
 import os
 
 def merge_data():
+    """
+    Merge bexar_property_all.h5 with texas_corp_merged.h5 and return
+    bexar_preprocessed.h5
+    """
     # Load property dataframe
     bexar_property = pd.read_hdf('./data/raw_h5_files/bexar_property_all.h5')
```

Comment:
this file name should also be defined in yaml so this script could easily be run on new data